src/ontology/mondo-edit.obo

```diff
@@ -520167,6 +520139,52 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
+subset: gard_rare {source="GARD:640"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:3319"}
+subset: orphanet_rare {source="Orphanet:3319"}
+subset: rare
+synonym: "amegakaryocytic thrombocytopenia, congenital" EXACT [MONDO:Lexical, OMIM:604498]
+synonym: "CAMT1" EXACT ABBREVIATION [DOID:0090118, MONDO:Lexical, OMIM:604498, Orphanet:3319]
+synonym: "congenital amegakaryocytic thrombocytopenic purpura" EXACT [DOID:0090118, Orphanet:3319]
+synonym: "thrombocytopenia congenital amegakaryocytic" RELATED [GARD:0000640]
+synonym: "thrombocytopenia, congenital amegakaryocytic" EXACT [OMIM:604498, OMIM:genemap2]
+xref: DOID:0090118 {source="MONDO:equivalentTo"}
+xref: GARD:640 {source="Orphanet:3319"}
+xref: ICD10CM:D61.0 {source="Orphanet:3319/attributed", source="Orphanet:3319/ntbt", source="Orphanet:3319", source="DOID:0090118"}
+xref: MESH:C535982 {source="Orphanet:3319/e", source="MONDO:equivalentTo", source="Orphanet:3319", source="DOID:0090118"}
+xref: NCIT:C115207 {source="MONDO:equivalentTo"}
+xref: OMIM:604498 {source="Orphanet:3319/e", source="MONDO:equivalentTo", source="Orphanet:3319", source="DOID:0090118"}
+xref: Orphanet:3319 {source="OMIM:604498", source="MONDO:equivalentTo", source="DOID:0090118"}
+xref: SCTID:716336002 {source="MONDO:equivalentTo"}
+xref: UMLS:C1327915 {source="OMIM:604498", source="Orphanet:3319/e", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="Orphanet:3319", source="DOID:0090118", source="NCIT:C115207"}
+is_a: MONDO:0800451 {source="OMIM:604498", source="https://orcid.org/0000-0001-5208-3432"} ! congenital amegakaryocytic thrombocytopenia
+intersection_of: MONDO:0800451 ! congenital amegakaryocytic thrombocytopenia
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/7217 ! MPL
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+relationship: has_characteristic MONDO:0021140 {source="https://orcid.org/0000-0002-4142-7153"} ! congenital
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/7217 {source="MONDO:mim2gene_medgen"} ! MPL
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/4069" xsd:anyURI
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/4521" xsd:anyURI
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6622" xsd:anyURI
+property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/640/congenital-amegakaryocytic-thrombocytopenia" xsd:anyURI {source="GARD:0000640"}
```

Comment:
May also need to be removed based on decisions above whether these references to CAMT are relevant for CAMT1.